src/tests/buildings/green.ts

```diff
@@ -0,0 +1,113 @@
+import { expect } from "chai";
+import { GameEngine } from "../../framework/engine";
+import { PlayerState } from "../../framework/types";
+import { requireActivePlayer } from "../../game/helpers";
+import {
+  P1,
+  debugAction,
+  debugGotoNextTurn,
+  debugPlayCard,
+  debugPlayUnit,
+  makeDefaultGame,
+} from "../testhelper";
+
+describe("buildings", () => {
+  describe("Moss Sentinels", () => {
```

Comment:
Is this the faction name for green...? We should standardize whether we want to use these names around or not, because generally I think we only use colors ever atm

Reply:
Yeah, that's the faction name. I figured it was acceptable in the unit tests to use the full name, and since these are the first unit tests for a faction's units, I guess I got to set the precedent. :)